addons/mail/static/tests/helpers/mock_server.js

```diff
@@ -701,13 +703,15 @@ MockServer.include({
                 ['model', '=', 'mail.channel'],
                 ['message_type', '!=', 'user_notification'],
             ]);
+            const lastMessageId = Math.max(...messages.map(message => message.id));
```

Comment:
should also reduce here

Reply:
check `_mockMailChannelChannelFetched` it's already done there

Reply:
Math.max does not really work properly for this purpose if there is no message at all

Reply:
Also remove `last_message_id` from the result of `_mockMailChannelChannelInfo` in livechat since it is done here now.